application/browser/application.cc

```diff
@@ -193,7 +193,12 @@ void Application::SetWindowShowState<Manifest::TYPE_MANIFEST>(
 
   if (display_string == values::kDisplayModeFullscreen) {
     params->display_mode = blink::WebDisplayModeFullscreen;
-    params->state = ui::SHOW_STATE_FULLSCREEN;
+    // Set ui::SHOW_STATE_MAXIMIZED instead of ui::SHOW_STATE_FULLSCREEN
+    // because there is no way to exit from full screen mode.
+    // Refer to XWALK-4861 for more details.
+#if defined(OS_WIN)
+    params->state = ui::SHOW_STATE_MAXIMIZED;
```

Comment:
It should remain ui::SHOW_STATE_FULLSCREEN, and new ui elements shall be introduced for the fullscreen window..maybe a buttons sliding down when the user is moving the mouse over the top of the window or so..  @darktears wdyt?